AGENTS.md

```
@@ -0,0 +1,95 @@
+
+## Session Completion
+
+**When ending a work session**, complete the checklist below. Steps that mutate shared remotes (push, prune) or discard local state (stash drop) require **explicit user authorization** unless the user already granted push/session-close autonomy for this session.
+
+**MANDATORY WORKFLOW:**
+
+1. **File issues for remaining work** - Create issues for anything that needs follow-up
+2. **Run quality gates** (if code changed) - Tests, linters, builds
+3. **Update issue status** - Close finished work, update in-progress items
+4. **PUSH TO REMOTE** (only with user authorization for remote mutation):
+   ```bash
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)